src/openrct2/sprites.h

```diff
@@ -1208,13 +1208,34 @@ enum
     SPR_G2_GIGA_RC_POWERED_LIFT_1,
     SPR_G2_GIGA_RC_POWERED_LIFT_2,
     SPR_G2_GIGA_RC_POWERED_LIFT_3,
+    SPR_G2_GIGA_TRACK_VERTICAL,
+    SPR_G2_GIGA_TRACK_VERTICAL_TWIST = SPR_G2_GIGA_TRACK_VERTICAL + 12,
+    SPR_G2_GIGA_TRACK_BARREL_ROLL = SPR_G2_GIGA_TRACK_VERTICAL_TWIST + 12,
+    SPR_G2_GIGA_TRACK_HALF_LOOP = SPR_G2_GIGA_TRACK_BARREL_ROLL + 48,
+    SPR_G2_GIGA_TRACK_VERTICAL_LOOP = SPR_G2_GIGA_TRACK_HALF_LOOP + 16,
+    SPR_G2_GIGA_TRACK_VERTICAL_LOOP_FIX_1 = SPR_G2_GIGA_TRACK_VERTICAL_LOOP + 32,
+    SPR_G2_GIGA_TRACK_VERTICAL_LOOP_FIX_2,
+    SPR_G2_GIGA_TRACK_VERTICAL_LOOP_FIX_3,
+    SPR_G2_GIGA_TRACK_VERTICAL_LOOP_FIX_4,
```

Comment:
Wait what did you do to the loop sprites? There's not supposed to be additional sprites added, there should just be 32.

Reply:
Added a few extra, cutout, sprites to fill on the spaces that were outside the loops drawing boundry. They now show in the correct spot, but bounding boxes are wrong.

Reply:
Yeah, but adding extra sprites isn't the best way to do this ... I mean, it can work, but I'd rather keep the sprite count and paint code the same as the vanilla rides because that would make a future refactor simpler ... and also, adding new sprites means more bounding boxes to tweak.

The way to fix the clipping is to move the parts of the sprites that clip to other sprites that won't, to match the way the vanilla loop sprites are split. I couldn't figure out how to get my renderer to output the correct splitting automatically, like it usually would - this isn't the first track piece that had to be split manually like this I just normally wouldn't submit a PR with such an issue. I'm going to try and fix this properly now. If I'm correct, this is the only thing left for the giga track?